backend/resources/users.py

```diff
@@ -81,6 +82,11 @@ def post(self):
                 user = UsersModel(**data)
                 user.hash_password(password)
                 user.save_to_db()
+
+                url_root = 'www.test.com/'
+                verify = VerifyModel(user.id)
+                verify.save_to_db()
+                verify.send_email(user.email, url_root)
```

Comment:
Utlitza la funcio, que et dona la url de forma dinamica:
```suggestion
                verify = VerifyModel(user.id)
                verify.save_to_db()
                verify.send_email(user.email, request.url_root)
```
Import necessari:
```python
from flask import request
```